charts/traefik/1.0.0/test_values.yaml

```diff
@@ -353,15 +373,25 @@ ports:
     # Set TLS at the entrypoint
     # https://doc.traefik.io/traefik/routing/entrypoints/#tls
   unificom:
-    port: 8081
+    port: 8080
```

Comment:
The tests have a tendency to go strangely haywire when using lowish portnumbers (see the weird config on some "ports" settings in this file.) they are fine for values.yaml though.

Reply:
If the ports are configured in the app's `appAdditionalServices`, does `unificom` even need to exist in `traefik`?

Reply:
@whiskerz007 Yes, what you are creating here is an 'entrypoint'. You tell Traefik to open a port to the outside world and handle traffic for it.

What "appAdditionalServices" does, is open a port on a "services"... Simply put an internal loadbalancer within k8s.